modules/local/treesummarizedexperiment.nf

```diff
@@ -0,0 +1,82 @@
+process TREESUMMARIZEDEXPERIMENT {
```

Comment:
This all looks pretty straight forward, any reason why this is not an official nf-core module?

Reply:
> any reason why this is not an official nf-core module?

First we needed to find a usable implementation, this is now done, now it seems just laziness on my part, but we can call it time efficiency or priorities are on other stuff or just 24h/day instead of 240h/day.